### ISSUE TITLE
Add the detail when I click on the top counter

### ISSUE THREAD

**Issue by pkriko** [centreon/centreon#7546]
<h1>BUG REPORT INFORMATION </h1>

<h3> Prerequisites </h3>

> The opened issue, must be code related. GitHub is not meant for support. Feel free to check the CONTRIBUTING section for more details.

***Versions***

For the RPM based systems

-- Copy/Paste the result of the following command --
```
$ rpm -qa | grep centreon
centreon-release-19.04-1.el7.centos.noarch
centreon-connector-ssh-19.04.0-2.el7.centos.x86_64
centreon-widget-live-top10-cpu-usage-19.04.0-5.el7.centos.noarch
centreon-clib-19.04.0-1.el7.centos.x86_64
centreon-19.04.0-3.el7.centos.noarch
centreon-map-web-client-19.04.1-9.el7.centos.noarch
centreon-engine-extcommands-19.04.0-2.el7.centos.x86_64
centreon-database-19.04.0-3.el7.centos.noarch
centreon-pp-manager-19.04.0-3.el7.centos.noarch
centreon-widget-host-monitoring-19.04.0-5.el7.centos.noarch
centreon-widget-live-top10-memory-usage-19.04.0-5.el7.centos.noarch
centreon-bam-release-19.04-1.el7.centos.noarch
centreon-perl-libs-19.04.0-3.el7.centos.noarch
centreon-widget-hostgroup-monitoring-19.04.0-5.el7.centos.noarch
centreon-mbi-release-19.04-1.el7.centos.noarch
centreon-trap-19.04.0-3.el7.centos.noarch
centreon-connector-perl-19.04.0-2.el7.centos.x86_64
centreon-license-manager-19.04.0-1.el7.centos.noarch
centreon-widget-tactical-overview-19.04.0-7.el7.centos.noarch
centreon-widget-servicegroup-monitoring-19.04.0-7.el7.centos.noarch
centreon-broker-storage-19.04.0-2.el7.centos.x86_64
centreon-widget-httploader-19.04.0-5.el7.centos.noarch
centreon-awie-19.04.0-1.el7.centos.noarch
centreon-fingerprint-19.04.0-1.el7.centos.x86_64
centreon-common-19.04.0-3.el7.centos.noarch
centreon-engine-daemon-19.04.0-2.el7.centos.x86_64
centreon-engine-19.04.0-2.el7.centos.x86_64
centreon-poller-centreon-engine-19.04.0-3.el7.centos.noarch
centreon-auto-discovery-server-19.04.0-4.el7.centos.x86_64
centreon-cwrapper-perl-0.1-1.el7.centos.x86_64
centreon-bi-server-19.04.0-1.el7.centos.noarch
centreon-widget-graph-monitoring-19.04.0-5.el7.centos.noarch
centreon-license-manager-common-19.04.0-1.el7.centos.noarch
centreon-web-19.04.0-3.el7.centos.noarch
centreon-widget-engine-status-19.04.0-5.el7.centos.noarch
centreon-widget-global-health-19.04.0-5.el7.centos.noarch
centreon-map-release-19.04-1.el7.centos.noarch
centreon-connector-19.04.0-2.el7.centos.x86_64
centreon-widget-grid-map-19.04.0-5.el7.centos.noarch
centreon-broker-core-19.04.0-2.el7.centos.x86_64
centreon-broker-cbd-19.04.0-2.el7.centos.x86_64
centreon-widget-service-monitoring-19.04.0-5.el7.centos.noarch
centreon-broker-19.04.0-2.el7.centos.x86_64
centreon-broker-cbmod-19.04.0-2.el7.centos.x86_64
centreon-base-config-centreon-engine-19.04.0-3.el7.centos.noarch
centreon-bam-server-19.04.0-3.el7.centos.noarch
```

***Operating System***
CentOS 7.5

***Additional environment details (AWS, VirtualBox, physical, etc.):***

<h3> Description </h3>

Add the detail about the volumetry when I click on the top counter.

<h3> Steps to Reproduce </h3>

Please describe precisely the steps to reproduce the encountered issue.

1. I logged in Centreon
2. I click on the top counter to get more informations about the volumetry 

<h3> Describe the received result </h3>

![image](https://user-images.githubusercontent.com/32265250/57926630-109ddd00-78ac-11e9-8658-593a6ab4260a.png)

![image](https://user-images.githubusercontent.com/32265250/57926642-185d8180-78ac-11e9-95e0-0199e9c1f90e.png)

<h3> Describe the expected result </h3>
I would like to have the detail about my volumetry. Not 2K hosts, but the 1546 hosts. The same for the services.
<h3> Additional relevant information (e.g. frequency, ...) </h3>

**Comment by lpinsivy** [comment]
Hi @pkriko what about displaying 1567893 / 2500000?
It's not so human readable, isn't it?

**Comment by pkriko** [comment]
Hello Laurent,

I understand your point.

Can we get at least the global volumetry for the "All" line/ and the OK ? 

And Also on my screenshot, I have 1K of host is the Top counter but 2K in the detail of my host.

**Comment by cgagnaire** [comment]
Hi @lpinsivy,
We see your point but what about rounding 2501 to 3k ?
We have too find a proper mix between.

**Comment by kduret** [comment]
fixed on 19.10.0
